### PR TITLE
Issue #286

### PR DESCRIPTION
**Pull request by geirkairam**
add threshold date functions

### CODE REVIEW

File: Client/MainWindowViewModel.cs [hunk 1]
```diff
@@ -5,18 +5,22 @@
...
 using System.Diagnostics;
 using System.ComponentModel;
 using System.Text.RegularExpressions;
 using System.Globalization;
+using Clipboard = System.Windows.Clipboard;
+using Control = System.Windows.Controls.Control;
+using KeyEventArgs = System.Windows.Input.KeyEventArgs;
+using MessageBox = System.Windows.MessageBox;
+using OpenFileDialog = Microsoft.Win32.OpenFileDialog;
+using SaveFileDialog = Microsoft.Win32.SaveFileDialog;
```
Comment (benrhughes) [inline Client/MainWindowViewModel.cs]: It's not immediately obvious to me why we need to import these winforms components?

Comment (geirkairam) [reply Client/MainWindowViewModel.cs]: I removed those imports.

**Comment by geirkairam** [comment]
Hey, do you need anything else to merge this?

**Comment by benrhughes** [comment]
Sorry, forgot this one was still open!

**Comment by shulist** [comment]
In testing 3.3.1 I am assuming that the Threshold date changes were implemented as per the **ReadMe.md**: ```
- Ctrl+Up: increase threshold date by 1 day
- Ctrl+Down: decrease threshold date by 1 day
- Ctrl+Left/Right: remove threshold date 
- Ctrl+S: set threshold date 
- Ctrl+Alt+P: add days to threshold date 
```However in testing this functionality it appears that nothing is working to spec.

**Comment by aubreyz** [comment]
To @shulist 
What confused me initially is that on the beta page https://github.com/benrhughes/todotxt.net/releases/tag/3.3.1-beta

the one download is in fact not a beta, but the other is even though both are under the header "Pre-release of 3.3.1"

That's probably why you see it as not working

**Pull request merged**